test/src/main/java/org/apache/accumulo/test/compaction/CompactionExecutorIT.java

```diff
@@ -180,6 +212,18 @@ public static void teardown() {
 
   @AfterEach
   public void cleanup() {
+    var iter = compactionGroups.iterator();
+    while (iter.hasNext()) {
+      var group = iter.next();
+      getCluster().getConfig().getClusterServerConfiguration().addCompactorResourceGroup(group, 0);
```

Comment:
If its quick to explain, I am curious about why this is called with zero. If its not quick to explain let me know and I can look at the code.

Reply:
We set the value to zero so the next call to `getCluster().getClusterControl().start(ServerType.COMPACTOR);` doesn't start up old compactors.

Reply:
The other reason why we might have a group set to zero is because the miniAccumuloClusterImpl goes through on `start()` and creates compactorResourceGroups with a single compactor for each "group" it can find in the "executors" config.

However, there might be cases where we don't want that compactor to run by default, so we override that with a `0` setting. 

https://github.com/apache/accumulo/blob/feadf10305fec490a32211606d87a06a5b398d65/minicluster/src/main/java/org/apache/accumulo/miniclusterImpl/MiniAccumuloClusterImpl.java#L638-L642